src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2720,7 +2750,7 @@ public static Integer getNativeTransportMaxAuthThreads()
 
     /**
      * If this value is set to <= 0 it will move auth requests to the standard request pool regardless of the current
-     * size of the {@link org.apache.cassandra.transport.Dispatcher#authExecutor}'s active size.
+     * size of the {@link org.apache.cassandra.transport.Dispatcher}'s in authExecutor active size.
```

Comment:
opportunistic fix, this drives me crazy in IDEA as it marks that class as errorneous.